src/test/polymer-project_test.ts

```diff
@@ -78,10 +80,18 @@ suite('PolymerProject', () => {
     assert.isTrue(getFlowingState(dependencyStream));
   });
 
-  test('the bundler method returns a different bundler each time', () => {
-    const bundlerA = defaultProject.bundler();
-    const bundlerB = defaultProject.bundler();
-    assert.notEqual(bundlerA, bundlerB);
+  suite('.bundler()', () => {
+
+    test('returns a different bundler each time', () => {
+      const bundlerA = defaultProject.bundler();
+      const bundlerB = defaultProject.bundler();
+      assert.notEqual(bundlerA, bundlerB);
+    });
+
+    test('takes options to configure bundler', () => {
```

Comment:
I guess this description was a hold-over from when it took a bunch of options and they were defined.  I've updated it to show it takes the actual options now.